docs/pages/concepts/offloading/resource-reflection.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Resource reflection
+weight: 4
+---
+
+In addition to Node and Pods lifecycle handling, the Liqo virtual kubelet implements a feature we call *resource reflection*.
+It is responsible for the propagation and synchronization of selected control plane information into remote clusters, to enable the seamless execution of offloaded Pods. Currently, the virtual kubelet supports the reflection of the following Kubernetes resources:
+
```

Comment:
Not clear to me.
Shouldn't we say that also _pods_ are reflected in the remote cluster?